packages/grafana-ui/src/components/FormField/_FormField.scss

```diff
@@ -0,0 +1,12 @@
+.form-field {
```

Comment:
is this a  copy of gf-form class?

Reply:
Yes, I followed @dprokop proposal of moving style that is used by the component.

Reply:
yes,  moving. This is copying. But maybe this is ok in this case. not sure